src/content/en/2021/compression.md

```diff
@@ -165,7 +165,7 @@ From these results we can see that, [*compared to 2020*](../2020/compression#fir
 
 Compression level is a parameter given to the encoder adjusting the amount of effort the encoder is applying to find redundancy in the input, to consequently achieve higher compression density.
 
-Brotli encoding allows compression levels from 0 to 11, while Gzip uses levels 1 to 9 as well as higher optimal compression such as with Zopfli, indicated `opt` in the graph below. A higher compression level results in slower compression, but does not substantially affect the decompression speed, even making it slightly faster.
+Brotli encoding allows compression levels from 0 to 11, while Gzip uses levels 1 to 9. On top of that, the Zopfli compressor provides a higher level of compression for Gzip, indicated `opt` in the graph below. A higher compression level results in slower compression, but does not substantially affect the decompression speed, even making it slightly faster.
```

Comment:
I think the mention of Zopfli and Gzip is a bit confusing since they ultimately are "Gzip" files (at least that is my understanding - but you may know better!). I think the suggested wording here is an improvement for that, but not sure if we could add more on this to make it even clearer? Either here or elsewhere in the text? It was one of the things I was going to ask about when reading the original text.

Reply:
I tried to reword it to: `Higher levels can be achieved for Gzip as well, with a tool such as Zopfli. This is indicated as `opt` in the graph below.`